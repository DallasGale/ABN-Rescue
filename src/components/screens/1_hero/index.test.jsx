/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import Hero from '.'

describe('<Hero />', () => {
  it('renders without crashing', () => {
    const component = render(<Hero />)
    expect(component).toMatchSnapshot()
  })
})
