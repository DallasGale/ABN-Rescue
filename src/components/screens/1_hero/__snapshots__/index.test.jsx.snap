// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styled__StyledContent-sc-1uo0d0q-0 dvAUAo"
      >
        <div
          id="trigger"
          style="position: relative; top: 500px;"
        />
        <h1
          class="styled__StyledTitle-sc-1uo0d0q-1 dxbPar"
          style="color: rgb(255, 255, 255);"
        >
          Ambition Wanted.
        </h1>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styled__StyledContent-sc-1uo0d0q-0 dvAUAo"
    >
      <div
        id="trigger"
        style="position: relative; top: 500px;"
      />
      <h1
        class="styled__StyledTitle-sc-1uo0d0q-1 dxbPar"
        style="color: rgb(255, 255, 255);"
      >
        Ambition Wanted.
      </h1>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
